client/src/components/Landing.style.js

```diff
@@ -41,11 +41,11 @@ export const H2 = styled.h2`
 
 export const ButtonSignUp = styled.button`
   margin: 0.5rem;
-  width: 243px;
-  height: 46px;
+  width: 15rem;
+  height: 3rem;
   background: var(--white);
-  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-  border-radius: 20px;
+  box-shadow: 0rem 4rem 4rem rgba(0, 0, 0, 0.25);
```

Comment:
I think all of these can be px since they're just box shadow! Changing from px to rem like this makes them huge 😆